src/org/ets/research/nlp/corenlp/zeromq/Server.java

```diff
@@ -63,7 +63,7 @@ public static void main(String[] args)
         {
                 
             // get the number of threads as an integer
-            threads = Integer.parseInt(args[1]);
+            int threads = Integer.parseInt(args[1].trim());
```

Comment:
Oh 💩 sorry, I missed this 😅  Should you do any Java in the future, Eclipse is a beast but I'm like lost without it at this point 😬 👍

Reply:
Yeah, I should have used it. Bad Jeremy. But it's okay now!